build.gradle

```diff
@@ -11,9 +11,16 @@ repositories {
 
 dependencies {
     compile     'joda-time:joda-time:2.9.9',
-                'com.google.inject:guice:4.0'
+                'com.google.inject:guice:4.0',
+                'com.amazonaws:aws-java-sdk-sqs:1.11.277',
+                'com.amazonaws:aws-java-sdk-kms:1.11.277',
+                'com.amazonaws:aws-encryption-sdk-java:1.3.1',
+                'com.fasterxml.jackson.datatype:jackson-datatype-joda:2.8.10',
+                'cloud.localstack:localstack-utils:0.1.9'
```

Comment:
We need it for running in local environment for acceptance test (to check Hub components send encrypted events to SQS in localstack). If it is not required, I am happy for this library to be moved to testCompile.

Reply:
I think that if some other library needs it, then it should import it explicitly, rather than pulling it in via this library.

(ie, in this library, we only use localstack in tests, so should add it to testCompile. If someone else wants to use it in their tests, then they should add it to their own testCompile.)

Reply:
Hub components running in local environment will not be able to use localstack library from testCompile. In this case, it will have to use localstack library from compile. I have moved localstack library to testCompile. This means that Hub components will need to pull in localstack library directly if it needs the library.